moveit_ros/moveit_servo/src/realtime.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright 2022 PickNik Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
My other comment pointed out that this shouldn't be copied. But if you have to copy it, you'll need to update the license file and copyright notices (at the project's root) because this is Apache, and the rest of MoveIt is BSD. The are compatible license so both can exist in a project, but you will need to make sure they are both called out at the root of the project correctly. One thing you must do to comply with is place a copy of the license file itself in the repo, not just this notice.